### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by carakessler**
1. Fix runCommand so it captures all of stdout, rather than just last line
2. Export function renderEmptyResult so other reporters can use it
3. TaskReporter takes in array of Results, not just one Result
4. Adding additional helper used a lot in an upgrade of another plugin

### CODE REVIEW

File: packages/core/src/data/builders.ts [hunk 1]
```diff
@@ -149,6 +149,21 @@ export function buildLintResultDataItem(
   };
 }
 
+export function buildLintResultsFromEslintOrTemplateLint(
```
Comment (scalvert) [inline packages/core/src/data/builders.ts:152]: A bit wordy of a name?

**Review by scalvert — APPROVED** [review]

File: packages/core/src/data/builders.ts (hunk 1 above)
Comment (carakessler) [reply packages/core/src/data/builders.ts:152]: we can just have it be `buildLintResults` (because the params are obvious) but felt like that wasnt descriptive enough
